examples/react-native/package.json

```diff
@@ -50,6 +50,7 @@
     "metro-react-native-babel-preset": "^0.70.2",
     "react-dom": "17.0.2",
     "react-native-dotenv": "^3.4.0",
+    "react-native-launch-arguments": "^3.1.4",
```

Comment:
In-app variables access is tricky: https://wix.github.io/Detox/docs/guide/launch-args#in-app-arguments-access
`react-native-launch-arguments` handles both ios and android.